lib/cc/engine/fingerprint.rb

```diff
@@ -33,7 +33,13 @@ def compute
       attr_reader :path, :cop_name, :message
 
       def stripped_message
-        message.gsub(/ \[.+\]$/, "")
+        # example full message with details:
+        # Metrics/MethodLength: Method has too many lines. [18/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)
+        url_regex = %r{ \(https\://github\.com.+\)}
```

Comment:
Looks like it's an issue attribute that comes from
https://github.com/bbatsov/rubocop/blob/master/config/enabled.yml#L5

I don't think it would hurt to make the regex more aggressive though. I can't think of any false positives involving urls in description.